tests/test_cli.py

```diff
@@ -383,12 +382,6 @@ def test_generate_hashes_with_editable():
             ],
         )
     expected = (
-        '#\n'
-        '# This file is autogenerated by pip-compile\n'
-        '# To update, run:\n'
-        '#\n'
-        '#    pip-compile --generate-hashes --output-file requirements.txt requirements.in\n'
-        '#\n'
```

Comment:
Couldn't this be left/adjusted?

Reply:
I think it's possible. On the other hand this header has already been tested in `test_write_header`. Do you think it is necessary to test it once again?

Reply:
Not really.

Just wondered first why it was removed - but that was not due to it not being there then anymore.

Reply:
Thank you for the feedback 👍

Reply:
You're welcome!  Thanks for working on pip-tools! :)